packages/cluster/src/defaults.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    dhis2Version: '',
+    image: 'dhis2/core:{version}-latest-alpine',
```

Comment:
Actually this will work (for `dev` and `2.32`) currently since they're currently set up to build automatically on docker hub.  Released versions (`2.32.0-alpine`) won't work though

Reply:
If this were `dhis2/core:{version}-alpine` it would work as `d2 cluster up 2.32-latest` and `d2 cluster up 2.32.0`, however it's a change from the previous behavior which would support `d2 cluster up 2.32` and `d2 cluster up 2.32.0`.  I think that's an OK change, but will require a major version bump I think

Reply:
Alternatively we could transform `2.32` into `2.32-latest` (also for dev and other branches) in the `up` code.

Also worth noting - `2.32-latest` won't find a db to seed, currently, but existing behavior already fails for `2.32.0`

Reply:
> Actually this will work (for `dev` and `2.32`) currently since they're currently set up to build automatically on docker hub. Released versions (`2.32.0-alpine`) won't work though

Yeah, in that case you would have to pass in a full image reference: `--image dhis2/core:2.32.0-alpine`.

I want to let https://github.com/dhis2/notes/issues/48 get settled, then we can figure out the patterns to support.

Reply:
Reverted to use amcgee/dhis2-core for now. Will tackle the official images in another PR.